vortex/core/factory.py

```diff
@@ -113,18 +114,34 @@ def create_dataset(dataset_config : EasyDict,
     else:
         raise TypeError('Unknown dataset "stage" argument, got {}, expected "train" or "validate"'%stage)
 
-    return DatasetWrapper(dataset=dataset, stage=stage, preprocess_args=preprocess_config,
+    if wrapper_format=='default':
+        return DefaultDatasetWrapper(dataset=dataset, stage=stage, preprocess_args=preprocess_config,
                           augmentations=augmentations, dataset_args=dataset_args)
+    elif wrapper_format=='basic':
+        return BasicDatasetWrapper(dataset=dataset, stage=stage, preprocess_args=preprocess_config,
+                          augmentations=augmentations, dataset_args=dataset_args)
+    else:
+        raise RuntimeError('Unknown dataset `wrapper_format`, should be either "default" or "basic", got {} '.format(wrapper_format))
 
 def create_dataloader(dataset_config : EasyDict, 
                       preprocess_config : EasyDict, 
-                      stage : str,
+                      stage : str = 'train',
                       collate_fn : Union[Callable,str,None] = None ):
 
     dataset_config = deepcopy(dataset_config)
     preprocess_config = deepcopy(preprocess_config)
 
-    dataset = create_dataset(dataset_config=dataset_config, stage='train', preprocess_config=preprocess_config)
+    dataloader_module = dataset_config.dataloader.dataloader
+
+    # For backward compatibility purpose
+    if dataloader_module=='DataLoader':
+        dataloader_module='PytorchDataLoader'
```

Comment:
in your proposal in #7, this is changed to `'DeafultLoader'`, so which one would it be best?

Reply:
subject to discuss, currently I change it to Pytorch data loader so that it is explicit and verbose compared to DALI loader

Reply:
I prefer `'PytorchDataLoader'` actually, since it is more verbose